### ISSUE TITLE
training for a classification target types

### ISSUE THREAD

**Issue by Mofahdi** [microsoft/mattergen#42]
None of the examples is shown for a classification type of target. Can you please post an example for classification like 1-metal and 0- non-metal?

I noticed in "mattergen.diffusion.model_target.py" that the developers take into account "logits" target types. I also saw the logit target types in "mattergen.diffusion.model_utils.py" in the function "convert_model_out_to_score". so how can I apply the logit target types in the package? do I have to edit a "conf" file?
I appreciate your help and thanks!

**Comment by ClaudioZeni** [comment]
Hi Mofahdi and thanks for reaching out.

The `space_group` conditioned model is an example of a model that has been fine-tuned on a discrete property. I would suggest starting from there to see how you can fine-tune (or train) a model on a classification-type task.
